test/cachex/actions/fetch_test.exs

```diff
@@ -95,4 +95,32 @@ defmodule Cachex.Actions.FetchTest do
     assert(result8 == { :error, :invalid_fallback })
     assert(result9 == { :error, :invalid_fallback })
   end
+
+  # This test ensures that the fallback is executed just once when a
+  # fallback commit and another fetch on the same key occur simultaneously.
+  test "fetching and committing the same key simultaneously from a fallback" do
+    for duration <- 1..10 do
```

Comment:
Does this set of test code really need to run 10 times? I know it's because of the change in duration, but can we not just pick `50` as a duration (or w/e), since it should be the same behaviour?

Reply:
Because the test doesn't always fail even when set up carefully, I run it 10 times to increase the chance of the wrong implementation failing the test. Even when run 10 times, it doesn't always fail on some machines. But the duration might not need to change, so I will investigate and maybe tweak this.

Reply:
Ah, I see. Is it possible to lift the test you had in that repo you linked me to? I don't mind if the test takes a second or two.

Reply:
This test basically does the same thing as what was in the test repo, but is compatible with old versions of Elixir. Since you don't mind it running longer, I'll see if I can decrease its false successes by raising the number of iterations.